app/templates/power-ups/client-library.html

```diff
@@ -124,14 +124,22 @@ <h3 id="data-methods">Data Methods</h3>
 <h3 id="capability-options">Capability Options</h3>
 <p>Each capability that you include in your initialization will take a reference to the <strong>t</strong> object and a options or context object. This object will include various things depending on the capability.</p>
   <p>Board Capability Options</p>
-  <pre>{
-  "context":{
-    "board":"55db14fd3e105ac84105bd75",
-    "command":"board-buttons",
-    "plugin":"564ddf493f183488ea5ddc0e"
+  <pre>
+"{
+  "context": {
+    "member": "556c8537a1928ba745504ad8",
+    "permissions": {
+      "board": "write",
+      "organization": "write",
+      "card": "write"
```

Comment:
Yeah, you're right. Also not seeing organization when the command is board-buttons.